FreeRTOS/iot-reference-arm-corstone3xx
# Add generated device certificate, device private key and code signing key usage to aws-iot-example

Add generated device certificate, device private key and code signing key usage to aws-iot-example

Description
-----------
The device device certificate and private key can be generated with a Python script. The generated keys are embedded into a separate binary blob that can be loaded to a predefined memory address. The provisioning procedure uses the data from this binary.

The code signing key is included in the provisioning bundle which is generated during the TF-M. This provisioning bundle also has to be loaded to a predefined address and during the MCUBoot initialization, the key is provisioned into the OTP.
The new TF-M version enables code-signing key generation but at the moment it's disabled for the aws-iot-example. 

Test Steps
-----------
Built the aws-iot-example and ran an OTA test.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
